### ISSUE TITLE
Feature Request: disable the Blue frame / circle around matching POIs

### ISSUE THREAD

**Issue by lintux** [plepe/OpenStreetBrowser#71]
Hi,
the blue circle or frame around a matching element to me is somehow confusing, having the pin/icon in parallel. I would either prefer to remove it or have this as optional (default no frame/circle).

BR
Stephan

**Comment by plepe** [comment]
Sorry for not replying earlier, I've been thinking about this for quite some time. Yes, I think you are right, the circle is not optimal. On the other hand I would be difficult to only show the geometry for non-nodes. My suggestion is, to use a dot instead. What do you think?

Current:
![blue_circle](https://user-images.githubusercontent.com/249012/53809426-ffa02980-3f54-11e9-8f65-f38cfeae3174.png)

Dot instead of circle:
![blue_dot](https://user-images.githubusercontent.com/249012/53809427-ffa02980-3f54-11e9-9b30-6c3ce2423945.png)

But when we are talking about changing this, maybe it would be a good idea to change the color as well?
Grey:
![grey_dot](https://user-images.githubusercontent.com/249012/53809428-0038c000-3f55-11e9-9523-730f23c74062.png)

Black:
![black_dot](https://user-images.githubusercontent.com/249012/53809424-ff079300-3f54-11e9-9d95-bc56b8c86e60.png)

Black, no border:
![black_noborder](https://user-images.githubusercontent.com/249012/53809425-ffa02980-3f54-11e9-8043-8144c65311dc.png)

Red (same color as marker):
![red_dot](https://user-images.githubusercontent.com/249012/53809722-97057c80-3f55-11e9-92a9-3ef91c2c17ba.png)

My favourite is the grey one.

Maybe we should also talk about a different color/style for the markers. But that's for another issue.

**Comment by lintux** [comment]
Hi, thanks for the reply -- I think I am fine with the dot approach and would also prefer grey as the colour for dot or the  shape of a marked way.

**Comment by plepe** [comment]
@Cybereric @IgorEliezer @severinmenard @debyos @waldyrious do you have any thoughts on this?

**Comment by IgorEliezer** [comment]
> do you have any thoughts on this?

I do. :)

It seems we are dealing with two different things with distinct functions: one is the **geopin**, which shows the filter output; the other is the **feature highlighter**, which highlights the feature itself (node, way, area) on the map.

I think it would look weird that, for way/area features you have both the geopin and the highlighter, while for node features you have only the geopin.

What could be done is to add a config option, such as "Show filter results with: geopin | map object highlighter | both (default)".

As to the color and the circle: since OSM map tends to be quite colorful in dense areas I prefer something that stands out on the map, that is, black or blue. The circle is better as the dot covers the POI icon, in my opinion.

**Comment by waldyrious** [comment]
I agree with @IgorEliezer. With the Black option above (and the Blue one, to a lessed degree), the highlight alone is sufficient to locate the features, so the pin seems excessive. I think the map is cleaner with just the highlighter so I'd welcome having the option to not show the pins.

Alternatively, we could have the pins show on hover (or on tap, for mobile).

On a related but separate note, I think the Grey, Red and Black (no border) options are way too washed out and lacking contrast to actually serve effectively as highlights.

**Comment by lintux** [comment]
Coming back to my initial reason to create the request: I think in areas with a lot of POIs close to each other, both elements (highlighted objects and Pins) create a confusing view. To enhance my initial proposal: maybe its an option to select pins or frames or both. For both I think a less colouful frame is looking better. for a standalone frame or circle it could be some more colour.

**Comment by IgorEliezer** [comment]
> (...) the highlight alone is sufficient to locate the features, so the pin seems excessive.

I agree with @waldyrious. Showing both is excessive. I'd change my config idea to "Show filter results with: geopin (default) | object highlighter".

Hypothetically, I wouldn't drop the the geopin feature since it makes the map query friendlier, in my opinion. Say, a townhall uses OSB as a city guide of some sort.

BTW, I showed OSB to some people at the townhall, they like it. :+1:

**Comment by plepe** [comment]
Interesting discussion. I think, the best would be to really test different approaches. Therefore I'm planning to set up a test environment for you. Stay tuned :-)

@IgorEliezer :
> As to the color and the circle: since OSM map tends to be quite colorful in dense areas ...

I would love to have a OSM background map without the icons. But I don't want to render a background map myself (because this would need a decent server).
